dcf/views.py

```diff
@@ -207,3 +209,18 @@ def dispatch(self, *args, **kwargs):
 class RobotsView(TemplateView):
     template_name = 'dcf/robots.txt'
     content_type = 'text/plain'
+
+def complaint(request, pk, slug):
```

Comment:
Please use CBV here.

Reply:
Also, make sure there only one complaint per user for item.
And let user update complaints.